hw/hw11/mailroom.py

```diff
@@ -0,0 +1,90 @@
+donors = {
+    'Homer': [25, 10, 20],
+    'Marge': [100, 50, 75],
+    'Bart': [5, 1, 2],
+    'Lisa': [5, 10, 12, 6]}
+
+
+def mailroom():
+    print("Welcome to Mailroom Madness\nChoose from the following:")
+
+    user_input = input("T - send a (T)hank You   R - create a (R)eport \nC - (C)reate file\
+    quit - Quit the program \n")
+
```

Comment:
You should think about converting your input to lower case here because that is what your conditions are checking for below.